source/features/repo-age.tsx

```diff
@@ -7,15 +7,41 @@ import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
-import fetchDom from '../helpers/fetch-dom';
-import {getRepoURL, looseParseInt} from '../github-helpers';
+import * as api from '../github-helpers/api';
+import {getRepoURL, looseParseInt, getRepoGQL} from '../github-helpers';
 
 const dateFormatter = new Intl.DateTimeFormat('en-US', {
 	year: 'numeric',
 	month: 'long',
 	day: 'numeric'
 });
 
+const getRepoAge = async (commitSha: string, commitCount: number): Promise<[string, string]> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			defaultBranchRef {
+				target {
+					... on Commit {
+						history(first: 5, after: "${commitSha} ${commitCount}") {
+							nodes {
+								committedDate
+								resourcePath
+							}
+						}
+					}
+				}
+			}
+		}
+	`);
+
+	const {committedDate, resourcePath} = repository.defaultBranchRef.target.history.nodes
+		.reverse()
+		// Filter out any invalid commit dates #3185
+		.find((commit: AnyObject) => new Date(commit.committedDate) > 0);
```

Comment:
What about this then?

```suggestion
		.find((commit: AnyObject) => new Date(commit.committedDate).getTime() > 0);
```

Reply:
Yup!